### ISSUE TITLE
installation on M1/arm64

### ISSUE THREAD

**Issue by fgr1986** [edgeimpulse/edge-impulse-cli#8]
npm installation fails under newer M1 archs:
```

npm ERR! code 1                                                                                                                                                                                          npm ERR! path /opt/homebrew/lib/node_modules/edge-impulse-cli/node_modules/sharp                                                                                                                                 npm ERR! command failed                                                                                                                                                                                  npm ERR! command sh -c (node install/libvips && node install/dll-copy && prebuild-install) || (node-gyp rebuild && node install/dll-copy)                                                                        npm ERR! CC(target) Release/obj.target/nothing/../node-addon-api/nothing.o                                                                                                                                       npm ERR!   LIBTOOL-STATIC Release/nothing.a                                                   npm ERR!   TOUCH Release/obj.target/libvips-cpp.stamp                                                                                                                                                            npm ERR!   CXX(target) Release/obj.target/sharp/src/common.o                                                                                                                                                     npm ERR! info sharp Downloading https://github.com/lovell/sharp-libvips/releases/download/v8.10.5/libvips-8.10.5-darwin-arm64v8.tar.brnpm ERR! ERR! sharp Prebuilt libvips 8.10.5 binaries are not yet available for darwin-arm64v8     
...vips8 not found
```

**Comment by janjongboom** [comment]
@fgr1986 Thanks a lot for noticing this. Will take a look.

**Comment by janjongboom** [comment]
Hi @fgr1986 Can you do this:

```
brew install vips
```

And reinstall? This should fix it, we're adding checks for M1 macs in the installation script in the next release.

**Comment by fgr1986** [comment]
Both install and reinstall fails on m1, with OS v11.2.2
```
> m1brew install vips
Error: wrong number of arguments (given 2, expected 1)
/opt/homebrew/Library/Homebrew/extend/os/mac/utils/bottles.rb:23:in `find_matching_tag'
/opt/homebrew/Library/Homebrew/utils/bottles.rb:112:in `fetch_checksum_for'
/opt/homebrew/Library/Homebrew/software_spec.rb:449:in `checksum_for'
/opt/homebrew/Library/Homebrew/software_spec.rb:407:in `tag?'
/opt/homebrew/Library/Homebrew/software_spec.rb:97:in `bottle_tag?'
/opt/homebrew/Library/Homebrew/software_spec.rb:101:in `bottled?'
/System/Library/Frameworks/Ruby.framework/Versions/2.6/usr/lib/ruby/2.6.0/forwardable.rb:230:in `bottled?'
/opt/homebrew/Library/Homebrew/formula.rb:348:in `bottle'
/opt/homebrew/Library/Homebrew/formula_installer.rb:193:in `install_bottle_for?'
/opt/homebrew/Library/Homebrew/formula_installer.rb:576:in `block in expand_requirements'
/opt/homebrew/Library/Homebrew/requirement.rb:242:in `block in prune?'
/opt/homebrew/Library/Homebrew/requirement.rb:240:in `catch'
/opt/homebrew/Library/Homebrew/requirement.rb:240:in `prune?'
/opt/homebrew/Library/Homebrew/requirement.rb:230:in `block (2 levels) in expand'
/System/Library/Frameworks/Ruby.framework/Versions/2.6/usr/lib/ruby/2.6.0/set.rb:338:in `each_key'
/System/Library/Frameworks/Ruby.framework/Versions/2.6/usr/lib/ruby/2.6.0/set.rb:338:in `each'
/System/Library/Frameworks/Ruby.framework/Versions/2.6/usr/lib/ruby/2.6.0/delegate.rb:83:in `method_missing'
/opt/homebrew/Library/Homebrew/requirement.rb:229:in `block in expand'
/opt/homebrew/Library/Homebrew/requirement.rb:228:in `each'
/opt/homebrew/Library/Homebrew/requirement.rb:228:in `expand'
/opt/homebrew/Library/Homebrew/formula.rb:1666:in `recursive_requirements'
/opt/homebrew/Library/Homebrew/formula_installer.rb:574:in `expand_requirements'
/opt/homebrew/Library/Homebrew/formula_installer.rb:520:in `compute_dependencies'
/opt/homebrew/Library/Homebrew/formula_installer.rb:211:in `verify_deps_exist'
/opt/homebrew/Library/Homebrew/formula_installer.rb:202:in `prelude'
/opt/homebrew/Library/Homebrew/cmd/install.rb:400:in `install_formula'
/opt/homebrew/Library/Homebrew/cmd/install.rb:315:in `block in install'
/opt/homebrew/Library/Homebrew/cmd/install.rb:313:in `each'
/opt/homebrew/Library/Homebrew/cmd/install.rb:313:in `install'
/opt/homebrew/Library/Homebrew/brew.rb:122:in `<main>'
fgarcia:tiger [~]  > alias m1brew
m1brew='arch -arm64e /opt/homebrew/bin/brew'
```

Let me know if I can provide further logs.

**Comment by fgr1986** [comment]
only arm64 arch is available, not aarch64:
```
fgarcia:tiger [~]  > brew install vips
Updating Homebrew...
==> Auto-updated Homebrew!
Updated 1 tap (homebrew/core).
No changes to formulae.

Error: Cannot install under Rosetta 2 in ARM default prefix (/opt/homebrew)!
To rerun under ARM use:
    arch -arm64 brew install ...
To install under x86_64, install Homebrew into /usr/local.
```

**Comment by janjongboom** [comment]
Thanks @fgr1986 I'll go and borrow an M1 Mac.

**Comment by janjongboom** [comment]
We've published v1.13 of the CLI that resolves this now!
